test/spec/MfaVerify_spec.js

```diff
@@ -1559,17 +1553,6 @@ function (Q, _, $, Duo, OktaAuth, LoginUtil, CryptoUtil, Util, MfaVerifyForm, Be
             });
           });
         });
-        itp('makes the right init request when rememberDevice is checked', function () {
-          return setupDuo({ 'features.rememberDeviceAlways': true }).then(function () {
-            expect($.ajax.calls.count()).toBe(2);
-            Expect.isJsonPost($.ajax.calls.argsFor(1), {
-              url: 'https://foo.com/api/v1/authn/factors/ost947vv5GOSPjt9C0g4/verify?rememberDevice=true',
-              data: {
-                stateToken: 'testStateToken'
-              }
-            });
-          });
-        });
```

Comment:
We still want to keep these tests since we are testing "rememberDevice=true" query param. The test just needs to be set up in a different way.